pkg/cmd/job/job_add.go

```diff
@@ -0,0 +1,25 @@
+package job
+
+import (
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+// newCmdClusterRun runs a given job right now.
+func newCmdJobAdd(ctx api.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: "add a job",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := metronomeClient(ctx)
```

Comment:
@kensipe : interesting, I hadn't thought about that. The CLI command is called `job` and not `metronome` so I can imagine there were some thoughts behind this decision.

Reply:
The thoughts behind this were that we wanted to make a metronome package in the pkg directory that could be imported by other projects. So I would keep the package name as is.

Reply:
Calling this helper function `jobClient` sounds ok, however the thing with `job` is that it's not a great name for a go package, from https://blog.golang.org/package-names : 

> Don't steal good names from the user. Avoid giving a package a name that is commonly used in client code. For example, the buffered I/O package is called bufio, not buf, since buf is a good variable name for a buffer.

And `job` is probably the most common variable name package callers would want to use.

Reply:
I would *NOT* keep the package as named metronome... we have a job service.  It is in the UI as job... it is a job.  metronome is the implementation of jobs and there is a talk about replacing it at some pt with another thing but that thing maintains the same v1 API of metronome.

Reply:
@kensipe : I agree that we don't call it metronome in the UI, even in the CLI the command is called `job` so we are fine on that aspect. However as a developer I think it makes sense to have a metronome client for the metronome project, the naming is just not exposed to the end-user.

In the future if we create another project which is compatible with the metronome API we could rename the package, or even keep it as is. As an analogy a mysql client can also work for MariaDB.

If you really really think it shouldn't be called `metronome` we should come-up with a good enough name, `job` is not an option for a Go package (see the link above), maybe `jobclient`.